.ci/acceptance-tests/inspec-post-approve.yml

```diff
@@ -7,7 +7,7 @@ image_resource:
     type: docker-image
```

Comment:
What's the difference in these images? Just new InSpec?

Reply:
And re running bundle install to re cache gems. Runs of this image were taking a while because it had to pull down new gems for everything

Reply:
cool.